packages/types/src/shared.ts

```diff
@@ -34,6 +34,32 @@ export enum UserFlags {
   ActiveDeveloper = 1 << 22,
 }
 
+export enum GuildMemberFlags {
+  /**
+   * Member has left and rejoined the guild
+   *
+   * @remarks
+   * This value is not editable
+   */
+  DidRejoin = 1 << 0,
+  /**
+   * Member has completed onboarding
+   *
+   * @remarks
+   * This value is not editable
+   */
+  CompletedOnboarding = 1 << 1,
+  /** Member is exempt from guild verification requirements */
```

Comment:
```suggestion
  /** Member is exempt from guild verification requirements, which allows a member who does not meet verification requirements to participate in a server. */
```
I don't think this is necessary, so feel free to just mark this as resolved.

Reference: https://discord.com/developers/docs/resources/guild#guild-member-object-guild-member-flags (the info box below)

Reply:
I feel like the discord one is already clear enough so this is isn't needed, feel free to say that i'm wrong